server/views/macros/summaryListWithChangeLinks.njk

```diff
@@ -0,0 +1,32 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% macro summaryListWithChangeLinks(array = [], missingValueText = 'Not entered', attributes = {}, classes='') %}
+  {% set rows = [] %}
+
+  {% for detail in array %}
+    {% if detail.html %}
+      {% set value = {html: detail.html | showDefault(missingValueText)} %}
+    {% else %}
+      {% set value = {text: detail.value | showDefault(missingValueText)} %}
+      {% set actions = { items: [{href: detail.changeLinkHref, text: "Change", visuallyHiddenText: detail.label, attributes: {"data-qa": "change-link"} }] } %}
```

Comment:
Introduced this because `summaryList.njk` only has a single `changeLinkHref` which all appears in each row, now one can pass `array.detail.changeLinkHref`